src/debugAdapter/goDebug.ts

```diff
@@ -246,7 +263,22 @@ class Delve {
 	}
 
 	close() {
-		this.debugProcess.kill();
+        if (this.debugProcess === null) {                                                    
+            this.call<DebuggerState>('Command', [{ name: 'halt' }], (err, state) => {
+                if (err) {
+                    console.error('Failed to halt.');
+                }
+                // else TODO:  Can't find a way to kill remote delve.
+                // {
+                //     this.call<number>('Detach', [true], (err, ret) => {
```

Comment:
I think it probably doesn't make sense to detach from the debuggee here - since the Code debugger didn't do the attach itself, it assumed the debug server was already running and attached to a process.